conf/igenomes.config

```diff
@@ -47,5 +47,24 @@ params {
       snpeffDb               = "GRCh38.86"
       vepCacheVersion        = "95"
     }
+    'GRCm38' {
+      acLoci                 = "${params.igenomes_base}/Mus_musculus/Annotation/ASCAT/fake.loci"
+      acLociGC               = "${params.igenomes_base}/Mus_musculus/Annotation/ASCAT/fake.loci.gc"
+      bwaIndex               = "${params.igenomes_base}/Mus_musculus/Sequence/BWAIndex/genome.fa.{amb,ann,bwt,pac,sa}"
```

Comment:
I notice that we don't have the same approach in the chipseq pipeline, maybe that's something that we want to harmonize at the nf-core level?
we should discuss about such matters on slack